cpp/ql/test/query-tests/Security/CWE/CWE-457/semmle/tests/test.cpp

```diff
@@ -192,6 +193,13 @@ void test16() {
   use(foo); // BAD (NOT REPORTED)
 }
 
+void test_va_copy(va_list va) {
```

Comment:
I put the test here since the preceding test is also about `va_list`. All the other tests are numbered consecutively, but I gave this one a name in order to avoid having to rename every test following this one.